### PR TITLE
Make Organizer Loadouts column clickable

### PR DESCRIPTION
**Pull request by robojumper**
GIF 11 02 2023 18-37-55

### CODE REVIEW

File: src/app/organizer/Columns.tsx [hunk 1]
```diff
@@ -482,7 +484,7 @@ export function getColumns(
       noSort: true,
       gridWidth: 'minmax(324px,max-content)',
       filter: (value) =>
-        typeof value === 'string' ? `perkname:${quoteFilterString(value)}` : undefined,
+        typeof value === 'string' ? `inloadout:${quoteFilterString(value)}` : undefined,
```
Comment (nev-r) [inline src/app/organizer/Columns.tsx:487]: should this really be changed?

Comment (robojumper) [reply src/app/organizer/Columns.tsx]: oops, good catch

File: src/app/organizer/Columns.tsx [hunk 2]
```diff
@@ -574,11 +576,19 @@ export function getColumns(
         const inloadouts = loadoutsByItem[item.id];
         return (
           inloadouts &&
-          inloadouts.length > 0 &&
-          inloadouts.map((l) => <div key={l.loadout.id}>{l.loadout.name}</div>)
+          inloadouts.length > 0 && (
+            <LoadoutsCell loadouts={inloadouts.map(({ loadout }) => loadout)} owner={item.owner} />
```
Comment (nev-r) [inline src/app/organizer/Columns.tsx:580]:
```suggestion
            <LoadoutsCell loadouts={inloadouts.map((l) => l.loadout)} owner={item.owner} />
```
think this is shorter and 2% less confusing

**Review by nev-r — APPROVED** [review]
working great ty

File: src/app/organizer/Columns.tsx [hunk 3]
```diff
@@ -602,6 +612,23 @@ export function getColumns(
   return columns;
 }
 
+function LoadoutsCell({ loadouts, owner }: { loadouts: Loadout[]; owner: string }) {
+  return (
+    <>
+      {loadouts.map((loadout) => (
+        <div
```
Comment (bhollis) [inline src/app/organizer/Columns.tsx:619]: I might prefer this to be an `a` element.

**Review by bhollis — APPROVED** [review]

**Pull request merged**